### PR TITLE
Restore "data-driven styling" row for all style properties

### LINKED ISSUE #529: Restore "data-driven styling" row for all style properties
So that there is no ambiguity, it's useful to explicitly indicate which properties _don't_ support DDS yet, in any SDK. We used to have a "data-driven styling" row for all style properties, with "Not yet supported" in all columns, but it got lost in #501. Let's restore it.

### PR DESCRIPTION
**Pull request by jfirebaugh**
Fixes #529

### CODE REVIEW

File: reference/v8.json [hunk 1]
```diff
@@ -1861,6 +1904,9 @@
       "sdk-support": {
         "basic functionality": {
           "js": "0.27.0"
+        },
+        "data-driven styling": {
+          "js": "Not supported"
```
Comment (anandthakker) [inline reference/v8.json]: I'm seeing this being treated like a version number in the generated docs:
screen shot 2017-01-26 at 7 09 21 pm

I think just removing the entry fixes it -- @jfirebaugh is there some other reason we need it in here?  If so we can tweak the html template to handle this...

**Review by anandthakker — CHANGES_REQUESTED** [review]

File: reference/v8.json (hunk 1 above)
Comment (jfirebaugh) [reply reference/v8.json]: Ah, whoops. Yeah, `fill-extrusion-opacity` is probably never going to support data-driven styling, for the reasons outlined in #545. I'll see if I can tweak the template to make this work.

Comment (jfirebaugh) [reply reference/v8.json]: Settling for a prose solution:

image

**Review by anandthakker — APPROVED** [review]
👍 LGTM

**Pull request merged**